pkg/schedule/coordinator.go

```diff
@@ -217,25 +296,25 @@ func (c *Coordinator) checkRegions(startKey []byte) (key []byte, regions []*core
 	}
 
 	for _, region := range regions {
-		c.tryAddOperators(region)
+		regionChan <- region
```

Comment:
What if the channel is full?

Reply:
In the current code, it will block here until patrol region workers receive regions.